packages/zmq-async/zmq-async.5.2.1/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Async-aware bindings to ZMQ"
+maintainer: ["Anders Fugmann <anders@fugmann.net>"]
+authors: ["Rudi Grinberg"]
+license: "MIT"
+homepage: "https://github.com/issuu/ocaml-zmq"
+bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.04.1"}
+  "zmq" {= version}
+  "async_unix" {>= "v0.11.0" & < "v0.16"}
+  "async_kernel" {>= "v0.11.0" & < "v0.16"}
+  "base" {>= "v0.11.0" & < "v0.16"}
```

Comment:
Is there something preventing it to be not available on v0.16?
```suggestion
  "async_unix" {>= "v0.11.0"}
  "async_kernel" {>= "v0.11.0"}
  "base" {>= "v0.11.0"}
```

Reply:
No. I will remove the upper bounds.